### PR TITLE
Registration form Web UI

### PR DESCRIPTION
**Pull request by srrathi**
Now this is complete site from my side only connection and data storage in firebase is remaining

### CODE REVIEW

**Comment by srrathi** [comment]
Firebase data updating is not working

File: app.js [hunk 1]
```diff
@@ -25,14 +25,15 @@ const q2Input = document.querySelectorAll(".ques2");
  
 
 
-form.addEventListener('submit', event => {
```
Comment (nsrCodes) [inline app.js]: This should be `submit`

File: index.html [hunk 2]
```diff
@@ -75,7 +120,7 @@
 						</tr>
 					</table>
 					<br>
-					<input type="submit" name="submit" value="submit">
+					<input type="SUBMIT" name="SUBMIT" value="SUBMIT" class="btn">
```
Comment (nsrCodes) [inline index.html]: Here type should be `submit`
Name and Value can be anything

**Review by nsrCodes — CHANGES_REQUESTED** [review]
Changing the Capitalisation of keyword `submit` and test your local branch again

**Comment by srrathi** [comment]
changes pushed

**Pull request merged**